app/lib/kanban.schema.ts

```diff
@@ -0,0 +1,35 @@
+
+export const COLUMNS = {
+  todo: { key: 'todo', label: 'Todo' },
+  doing: { key: 'doing', label: 'In Progress' },
+  done: { key: 'done', label: 'Done' }
+} as const;
```

Comment:
suggestion: use enum types that will support extending the columns or change it in future:
```suggestion
export enum KanbanStatus {
  Todo = 'todo',
  Doing = 'doing',
  Done = 'done'
}

export const COLUMNS: Record<KanbanStatus, { key: KanbanStatus; label: string }> = {
  [KanbanStatus.Todo]:   { key: KanbanStatus.Todo,   label: 'Todo' },
  [KanbanStatus.Doing]:  { key: KanbanStatus.Doing,  label: 'In Progress' },
  [KanbanStatus.Done]:   { key: KanbanStatus.Done,   label: 'Done' }
};
```